config.yaml.sample

```diff
@@ -430,6 +430,8 @@ batch:
   retry-attempts: 5
   # check store if metatile already exists, and if so skip processing
   check-metatile-exists: true
+  # used when enqueueing batch array jobs, can be "low" or "high"
```

Comment:
Can you add comment about what "low" and "high" mean?

Reply:
acc3153